README.md

```diff
@@ -170,6 +170,34 @@ jobs:
       - run: $GOPATH/bin/buildevents cmd $TRAVIS_BUILD_ID $STEP_SPAN_ID go-test -- go test -timeout 2m -mod vendor ./...
 ```
 
+## Attaching more traces from your build and test process
+
+Every command running through `buildevents cmd` will receive a `HONEYCOMB_TRACE` environment variable that contains a marshalled trace propagation context. This can be used to connect more spans to this trace.
+
+Ruby Beeline example:
+```ruby
+# at the very start of the command
+# establish a command-level span, linking to the buildevent
+process_span = Honeycomb.start_span(name: File.basename($PROGRAM_NAME), serialized_trace: ENV['HONEYCOMB_TRACE'])
+Honeycomb.add_field_to_trace('process.full_name', $PROGRAM_NAME)
+Honeycomb.add_field_to_trace('process.args', ARGV)
```

Comment:
This is an example, but I think we should be careful advocating adding cmd args as trace fields. Many apps provide sensitive data var command args which would be be leaked.

Reply:
I can remove it again, but fwiw, buildevents itself captures ARGV.